app/containers/paperShowActionBar/components/constant.tsx

```diff
@@ -0,0 +1 @@
+export const LAST_SUCCEEDED_EMAIL_KEY = "l_s_e_k";
```

Comment:
i think it's not proper location to here.
maybe `app/common/constants` or `app/constants` will be better.